Utils/Dataflow/test/utils/reconstruction.py

```diff
@@ -77,6 +123,15 @@ def check(chain, chain_data, task_id, indent=0):
     Recursively display the constructed chain starting from given task_id (so,
     the chain's root should be used for getting a full chain). Also, find
     matching chain_data for each task and display it as well.
+
+    :param chain: constructed chain
+    :type chain: dict
+    :param chain_data: chain_data of all tasks in the chain
+    :type chain_data: list
+    :param task_id: task from which to start
+    :type task_id: int
+    :param indent: indent, also can be used to measure the chain's length
+    :type indent: int
```

Comment:
Maybe specify that `indent` is some kind of the "recursion parameter"? I mean, to clarify that it is OK not to think about it when calling the function; or call it "initial indent" and say something about indentation in the description (or provide some example of the output).

I get that it is a debug function, yet as I try to understand the PR fully, I make everything to make my job easier and not to dig through the code every time I try to remember what is what ;)

Reply:
Right you are. Done.